src/storage_pool.cc

```diff
@@ -116,7 +117,16 @@ NAN_METHOD(StoragePool::LookupByUUID)
 NAN_METHOD(StoragePool::LookupByVolume)
 {
   Nan::HandleScope scope;
-  return;
+  auto hv = ObjectWrap::Unwrap<Hypervisor>(info.This())->handle_;
+  auto volume = ObjectWrap::Unwrap<StorageVolume>(info[0]->ToObject())->handle_;
+  
+  Worker::Queue(info[1], [=](Worker::SetOnFinishedHandler onFinished) {
```

Comment:
I understand the intention to reduce boilerplate, and I think a lot of that has been handled very well, I'm just saying when I looked at this initially my first reaction was "what the heck is info?". In my opinion boilerplate for pulling out values for later use with very readable names is acceptable, but I'm by no means like putting my foot down here. 

To the second point: yes, I totally think `Queue` could use a better name too. Right now it looks like a class definition imho, so something like `QueueWorker` or `QueueCallback` or `SparklePony` would help
</bikeshedding>